packages/web-components/src/components/link-list/link-list.ts

```diff
@@ -10,6 +10,7 @@ import { customElement, html, LitElement, property } from 'lit-element';
 import settings from 'carbon-components/es/globals/js/settings';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
 import sameHeight from '@carbon/ibmdotcom-utilities/es/utilities/sameHeight/sameHeight';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
```

Comment:
@IgnacioBecerra I think you cut from the master branch for this change, you might need to cut a branch off the release branch instead, because we don't want to have changes from the master branch for the release.